package.json

```diff
@@ -3,139 +3,96 @@
     "================================================================================",
     "# To work within the development environment, run the following tasks",
     "  1. $ npm run dev",
-    "  2. $ npm run serve",
-    "  3. Go to http://localhost:3001",
+    "  2. Go to http://localhost:8000",
     "================================================================================"
   ],
-  "name": "@alaskaairux/auro-table",
-  "version": "1.0.2",
-  "description": "Auro custom auro-table element",
-  "author": "Product design and research",
+  "name": "@aurodesignsystem/auro-table",
```

Comment:
I just looked at the commits, there is no `BREAKING CHANGE` reference in the commits. This needs to be a MAJOR release.